mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -65,7 +67,7 @@ const uint16_t arm_disarm = 1000;
 const uint16_t heartbeat = 1000;
 const uint16_t set_mode = 1000;
 const uint16_t takeoff = 1000;
-const uint16_t rotate = 1000;
+const uint16_t rotate = 200;
```

Comment:
Milliseconds, as the namespace says: request_intervals_ms::
Is that what you mean?